pycue/tests/cloudinstancegroup_test.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
```

Comment:
Regarding file naming/location: the convention we follow is that test classes and their directory structure should mirror the class under test.

In this case, the class under test is `opencue.cloud.gce_api.GoogleCloudGroup` so this file should be `pycue/tests/cloud/gce_api_test.py`.

You've named your test class `GoogleCloudGroupTest` which already follows the convention for test class names. 👍